openapi/schemas/Tag.yaml

```diff
@@ -0,0 +1,8 @@
+type: "array"
```

Comment:
Do we want this to be an array - what if we introduce a new endpoint later that only handles a single tag?

Reply:
Good point. it will not be an array in the Tag.yaml specification